packages/@o3r/design/schematics/design-token-to-component/index.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  apply,
+  MergeStrategy,
+  mergeWith,
+  move,
+  renameTemplateFiles,
+  Rule,
+  template,
+  url
+} from '@angular-devkit/schematics';
+import { createSchematicWithMetricsIfInstalled } from '@o3r/schematics';
+import { basename, dirname, relative } from 'node:path';
+import type { NgAddDesignTokenSchematicsSchema } from './schema';
+
+/**
+ * Add Design Token to an existing component
+ * @param options
+ */
+export function ngAddDesignTokenFn(options: NgAddDesignTokenSchematicsSchema): Rule {
+
+  const name = basename(options.path, '.design-token.json');
```

Comment:
Maybe we should specify in the doc that file should follow this naming convention
else they will have issue :)

Reply:
Or even better we can set a pattern validator on the schema